libs/core/src/components/pds-select/pds-select.scss

```diff
@@ -0,0 +1,127 @@
+:host {
+  --box-shadow-focus: 0 0 0 1px #ffffff, 0 0 0 3px var(--pine-color-purple-300);
+  --box-shadow-focus-error: 0 0 0 1px #ffffff, 0 0 0 3px var(--pine-color-red-300);
```

Comment:
Thanks for adding all these custom props. This will make them much easier to update when semantic tokens are fully addressed.